interview-questions.md

```diff
@@ -9,58 +9,87 @@ Then, research the question to expand on your answer. Even if you feel you have
 
   Your answer:
 
+  - GitHub is an online platform which allows many people to work off of it and collaborate on a project. Git is local software that is on your computer.
+
   Researched answer:
 
+  - GitHub is an online platform and graphical user interface that works for the local, git software. Git is software that is local and not online.
 
 
 2. What types of evaluations will return a Boolean value?
 
   Your answer:
 
+  - True or false questions
```

Comment:
Sure, but when can you expect to see these? Specifically we are looking for logical, equality, and relational operators.